extensions/adobe/experience/adcloud/conversionmodel.schema.json

```diff
@@ -0,0 +1,160 @@
+            "Product Target": "Product Target",
+            "Product Group": "Product Group",
+            "null": "Unknown"
+          }
+        },
+        "xdm:propertyName": {
+          "title": "Property Name",
+          "type": "string",
+          "description": "Property Name Configured by Pixel, Analytics, Custom Feeds"
+        },
+        "xdm:modelName": {
+          "title": "Model Name",
+          "type": "string",
+          "description": "Unique Name of the advertising attribution Model. Examples include Last Event, First Event, Even Distribution"
+        },
+        "xdm:clickThroughCount": {
+          "title": "Click Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Count for attributed models if trigger event is Click"
+        },
+        "xdm:clickThroughValue": {
+          "title": "Click Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Value for attributed models if trigger event is Click"
+        },
+        "xdm:viewThroughCount": {
+          "title": "View Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Count for attributed models if trigger event is Impression"
+        },
+        "xdm:viewThroughValue": {
+          "title": "View Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Value for attributed models if trigger event is Impression"
+        },
+        "xdm:clickviewThroughValue": {
+          "title": "Click View Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Value for attributed models if trigger event is click but impressions present in path"
+        },
+        "xdm:clickviewThroughCount": {
+          "title": "Click View Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Count for attributed models if trigger event is click but impressions present in path"
+        },
+        "xdm:viewclickThroughValue": {
+          "title": "View Click Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Raw Conversion Value for attributed models if trigger event is impression but clicks present in path"
+        },
+        "xdm:viewclickThroughCount": {
+          "title": "View Click Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Raw Conversion Count for attributed models if trigger event is impression but clicks present in path"
+        },
+        "xdm:conversionType": {
```

Comment:
Should this modeled as enum?

Reply:
conversion_type so far has been integer. 0 for pixel, 1 for feed, to name a few. A few more have been added for cross-deivce

Reply:
We can model them as enum but I would rather wait for that to ensure we have them well defined for house hold project.

Reply:
I have added them as enum